v1/6_security.md

```diff
@@ -1,13 +1,49 @@
 # Security
 
-## Denial of Service (DoS) attacks
+In order to keep the protocol secure, we need to mitigate the risks associated with malicious actors and the risk of losing funds.
+This section outlines the security measures in place to protect the integrity of the Coinswap protocol, the possible abort scenarios and the mechanisms to prevent them or penalize the malicious actors.
 
-A denial of service (DoS) attack is a type of attack where a malicious actor floods the network with a large number of requests, causing the network to become overwhelmed and unresponsive. In the context of Coinswap, a DoS attack could disrupt the swap process and prevent legitimate users from participating in the network.
+## Aborts
 
-## Distributed Denial of Service (DDoS) attacks
```

Comment:
Is there any reason for not mentioning about DOS attack?

Reply:
Defending against DoS would be a part of the extended infrastructure(setting up reverse proxies, etc) and not a part of the protocol itself.